### ISSUE TITLE
Question: possibility to connect 'locations'

### ISSUE THREAD

**Issue by dojo90** [skx/simple-vpn#13]
Let's asume you have the following setup:

![grafik](https://user-images.githubusercontent.com/17551597/53080303-a538af00-34f8-11e9-9ee8-b7376ab80511.png)

- Location 1,2,3 should not be able to talk to each other
- A client from Location 4 can connect/talk to one client from location 1-3
- Location 4 should get the status if a client from location 1-3 is connected (in this example location 2 is offline)

**Comment by skx** [comment]
That operation is not supported, and not something I'd consider worth supporting.

This VPN is designed specifically to allow multiple locations to cross-talk.  That's pretty much implicit because of the way we setup the routing - we route the whole shared network-range to all peers.  You could cripple connections by setting up routes as /32s, but that would get hard to manage.

I'd be tempted to say use firewalling, but that's also fragile for the same reason.

In short: The software cannot be setup to handle only partial connectivity, and I think the same is true for most of the simple alternatives such as `tinc`, etc.

**Comment by skx** [comment]
PS.  Great use of an image though, that made it 100% obvious what you meant :)

**Comment by dojo90** [comment]
Thank for your fast reply!

Actually we got a commercial solution but it costs too much and the performance is very bad.

> PS. Great use of an image though, that made it 100% obvious what you meant :)

Made it with mermaid in `vscode`. But theres also an online editor. see: https://mermaidjs.github.io/mermaid-live-editor
